packages/reaction-product-variant/client/templates/products/productDetail/variants/variantList/variantList.js

```diff
@@ -6,31 +6,38 @@ Template.variantList.helpers({
     let inventoryTotal = 0;
     const variants = ReactionProduct.getTopVariants();
 
-    if (variants.length > 0) {
+    if (variants.length) {
       // calculate inventory total for all variants
-      for (let variant of variants) {
-        let qty = ReactionProduct.getVariantQuantity(variant);
+      variants.forEach(variant => {
```

Comment:
The `for of` didn't need to be changed here. Also, I believe `Array.forEach` came with ES5 (JS 1.6), and `for of` is ES2015 (aka ES6). If you want to adopt a more ES6 like coding style, then you should be using `for of`.

Native array iterators are useful such as `find`, `every`, `map`, etc. In general `for of` will do everything the native `Array.forEach` method will, except give you the index. `for of` loops are also breakable.

tl;dr Prefer `for of` loops. Use the other native iterators as needed.

See other [comment here](https://github.com/reactioncommerce/reaction/pull/950/files#r58445986)